psqlgraph/mocks.py

```diff
@@ -223,6 +241,20 @@ def create(self, label, override=None, all_props=False):
     def get_global_value(self, prop):
         return self.graph_globals.get('properties', {}).get(prop)
 
+    def validate_override_value(self, prop, label, override):
+        # we allow specific passed values to override if they are valid
+        if not override:
+            return
+
+        override_val = override.get(prop)
+        try:
+            if self.property_factories[label].type_factories.get(
```

Comment:
small thing, but since you are catching `KeyError` you might just do `type_factories[prop]`

Reply:
sure no sweat